client/my-sites/stats/stats-period-navigation/index.jsx

```diff
@@ -52,21 +52,34 @@ class StatsPeriodNavigation extends PureComponent {
 
 	handleArrowNext = () => {
 		const { date, moment, period, url, queryParams } = this.props;
-
-		const usedPeriod = 'hour' === period ? 'day' : period;
```

Comment:
Yes, I believe so too - AFAIK this isn't breaking anything anywhere, and my implementation isn't likely to break the Email Stats page. However, I'm having trouble accessing the email stats page to test. Did you see any indication that this was breaking anything?

Reply:
Yes. I have tried this PR with the Email Stats page. And if I press the button to navigate next on the period `hour`, the operation is stuck. We can keep the original code since it would not break our pages.

##### Before
https://user-images.githubusercontent.com/6869813/213317227-17a28696-6a86-46eb-9ce8-9e051fc5773c.mov

##### This PR
https://user-images.githubusercontent.com/6869813/213317291-741a85fc-0da6-473a-8539-bcaafbb1d113.mov

Reply:
```
const usedPeriod = 'hour' === period ? 'day' : period;
```
Any particular reason to remove the line? @annacmc

Reply:
Ah, I see - sorry I misunderstood! Yes, you're right - it is needed for that page :)

Reply:
> Any particular reason to remove the line? @annacmc

Nope, sorry. Definitely a mistake!